demos/segmentation_demo/python/README.md

```diff
@@ -86,6 +86,14 @@ You can use the following command to do inference on CPU on images captured by a
     python3 segmentation_demo.py -i 0 -m <path_to_model>/semantic-segmentation-adas-0001.xml
 ```
 
+## Color palettes
+
+The color palette is used to visualize predicted classes. By default, the colors from PASCAL VOC dataset are applied. In case then the number of output classes is larger than number of classes provided by PASCAL VOC dataset, the rest classes are randomly colorized.
+Also, one can use predefined colors from other datasets, like CAMVID.
+
+Available colors files are in [palettes](../../../data/palettes).
```

Comment:
```suggestion
Available colors files are in '<omz_dir>/data/palettes`.
```
when openvino onlne documentation generated from markdown files it will not point to relative path